docs-site/content/en/docs/Deployment Options/vanilla-installation/index.md

```diff
@@ -3,109 +3,138 @@ categories: ["quickstart"]
 tags: ["test", "sample", "docs"]
 title: "Vanilla Installation"
 linkTitle: "Vanilla Installation"
-date: 2023-03-07
+date: 2025-08-19
+weight: 2
 description: >
   Deploy kubeflow into an AKS cluster using default settings.
 ---
 
 ## Background
 
-In this lab you will deploy an Azure Kubernetes Service (AKS) cluster and other Azure services (Container Registry, Managed Identity, Key Vault) with [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli) and [Bicep](https://docs.microsoft.com/azure/azure-resource-manager/bicep/overview). You will then install Kubeflow using the default settings using Kustomize and create a jupyter notebook server you can easily access on your browser.
+In this lab you will deploy an [Azure Kubernetes Service (AKS) Automatic](https://learn.microsoft.com/en-us/azure/aks/intro-aks-automatic) cluster with [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli). AKS Automatic provides a simplified, managed Kubernetes experience with automated node management, scaling, and security configurations. You will then install Kubeflow using the default settings using Kustomize and create a jupyter notebook server you can easily access on your browser. 
```

Comment:
What are the pros and cons of using AKS automatic vs regular AKS to assist the user in making this decision? I see the pros are mentioned. Are there cons like specific features may not be available in automatic?

Reply:
added notes